AppCenterData/AppCenterData/MSData.m

```diff
@@ -786,8 +786,8 @@ - (void)processPendingOperations {
                                       reachability:self.reachability
                                  completionHandler:^(MSTokensResponse *_Nonnull tokenResponses, NSError *_Nonnull error) {
                                    if (error) {
-                                     MSLogError([MSData logTag], @"Cannot read from local storage because there is no "
-                                                                 @"account ID cached and failed to retrieve token.");
+                                     MSLogWarning([MSData logTag], @"Cannot read from local storage because there is no "
```

Comment:
This seems a duplicate log for a single operation. We could eliminate one of them.

Reply:
Agree with @jaelim-ms. If `(void)performDbTokenAsyncOperationWithHttpClient:` already logs properly for each error case (https://github.com/microsoft/appcenter-sdk-apple/pull/1632/files#diff-a797f8f186af5727476c3ecc3ad4f2fbR29), then we can remove this line completely.

Reply:
I'd prefer to keep this as Warning since it provides more context around the call that ends up to an unwanted state. Token Exchange only logs generic errors.